00-intro/intro.md

```diff
@@ -11,4 +11,7 @@ For more information about the structure and expectations of the post-classroom
 
 ### Resources
 
-- [Post-Classroom CS Fundamentals Site](https://sites.google.com/adadevelopersacademy.org/post-classroom/unit-4/cs-fundamentals?authuser=1)
\ No newline at end of file
+- [Post-Classroom CS Fundamentals Site](https://sites.google.com/adadevelopersacademy.org/post-classroom/unit-4/cs-fundamentals?authuser=1)
+- [PSE Feedback Request Form](https://form.asana.com/?k=s_JorjEM5mBvzcDtfw8yJg&d=181459410160484)
+    - Please be aware grading is subject to the availability of instructor capacity. We will make every effort to evaluate your PSE within one week of receiving your completed form.
+    - Please submit one form per assignment.
```

Comment:
Could we add an additional point here advising to students to review video lessons in Learn with solutions or reviewing recorded PSE debriefs during instructor lead review sessions before requesting feedback through this form?

Maybe something like "Given limited instructor capacity, we ask that you first attend review sessions which will include a debrief of the topic's PSE before requesting feedback. As always, If you are not able to attend the review session live, we will record the sessions for you to watch back later."

Reply:
Thank you for the reminder 🙏🏾 Added a bullet point + a lil bit of additional information about what folx can access on the Google site.